api/kyverno/v1alpha1/cleanup_policy_types.go

```diff
@@ -194,7 +195,7 @@ func ValidateSchedule(path *field.Path, schedule string) (errs field.ErrorList)
 
 // ValidateMatchExcludeConflict checks if the resultant of match and exclude block is not an empty set
 func (spec *CleanupPolicySpec) ValidateMatchExcludeConflict(path *field.Path) (errs field.ErrorList) {
-	if len(spec.ExcludeResources.All) > 0 || len(spec.MatchResources.All) > 0 {
+	if spec.ExcludeResources == nil || len(spec.ExcludeResources.All) > 0 || len(spec.MatchResources.All) > 0 {
```

Comment:
Why return errors when exclude is nil? Isn't it optional?

Reply:
Errs is nil so it doesn't return errors, it just returns early

Reply:
Ah got it!